### PR TITLE
Record git compare in redis for each new release discovered.

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

**Review by james-jdgtl — APPROVED** [review]
nice!

File: health_ping.py [hunk 1]
```diff
@@ -20,6 +22,9 @@
 redis_max_stream_length = int(os.getenv("REDIS_MAX_STREAM_LENGTH", "360"))
 refresh_interval = int(os.getenv("REFRESH_INTERVAL","60"))
 log_level = os.environ.get('LOG_LEVEL', 'INFO').upper()
+GITHUB_APP_ID = int(os.getenv('GITHUB_APP_ID'))
+GITHUB_APP_INSTALLATION_ID = int(os.getenv('GITHUB_APP_INSTALLATION_ID'))
+GITHUB_APP_PRIVATE_KEY = os.getenv('GITHUB_APP_PRIVATE_KEY')
```
Comment (andrewrlee) [inline health_ping.py:27]: add these to `.env.example`?

File: .env.example [hunk 2]
```diff
@@ -3,4 +3,7 @@ SERVICE_CATALOGUE_API_KEY=
 REDIS_ENDPOINT=localhost
 REDIS_PORT=6379
 REDIS_TLS_ENABLED=false
-SERVICE_CATALOGUE_FILTER=&filters[name][$contains]=book-a-prison-visit-staff-ui
+SERVICE_CATALOGUE_FILTER=&filters[name][$contains]=
+GITHUB_APP_ID =
+GITHUB_APP_INSTALLATION_ID =
+GITHUB_APP_PRIVATE_KEY =
```
Comment (psoleckimoj) [inline .env.example:9]: Missing trailing newline

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**